documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+
+[id="non-admin-permissions_{context}"]
+== Granting permissions to non-administrators to work with migration plan components
+
+As an administrator, you can work with all components of migration plans (for example, providers, network mappings, and migration plans).
+
+By default, non-administrators have limited ability to work with migration plans and their components. You can modify their roles to allow them full access to all components or you can give them limited permissions.
+
+For example, you can grant non-administrators the following permissions:
+
+* Create and modify storage maps, network maps, and migration plans for the namespaces they have access to
+* View, but not create, providers and attach them to components as needed
+
+[cols="3", options="header"]
+.Permissions required for non-adminstrators to work with migration plan components but not create providers
+|===
+|Actions |API groups |Resources
+
+|All
+|`forklist.conveyer.io`
+|`plans`
+
+|All
+|`forklist.conveyer.io`
+|`migrations`
+
+|All
+|`forklist.conveyer.io`
+|`hooks`
+
+|`get`, `list`, `watch`
+|`forklist.conveyer.io`
+|`providers`
+
+|All
+|`forklist.conveyer.io`
+|`networkmaps`
+
+|All
+|`forklist.conveyer.io`
+|`storagemaps`
+
+|`get`, `list`, `watch`
+|`forklist.conveyer.io`
+|`forkliftcontrollers`
+|===
+
+.Procedure
+
+* To grant cluster-wide permissions, use `ClusterRoleBinding` to add the permissions previously described to a `ClusteRrole` and bind them to the user.
+
+* To grant permissions for only specific namespaces, use `RoleBinding` to add the following to the user's role:
+
+** The permissions previously described
+** `Get`permission for the namespaces you want to permit them to work with
```

Comment:
should the last two lines remain?

Reply:
Will replace lines 53-56 with the note you suggest below.

Reply:
I left line 51 in -- did you want it removed as well? I thought from the Jira that lines 51-56 were needed for different uses of the permissions.